portfolio/src/main/webapp/script.js

```diff
@@ -62,3 +62,67 @@ function changeProfilePic() {
   document.getElementById('pfp-link').href = '/images/' + img;
   document.getElementById('pfp').src = '/images/' + img;
 }
+
+let xando = [
+  [0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0],
+];
+
+/**
+ * Process tic-tac-toe game.
+ */
+function ticTacToe(cell) {
+  const playerTurn = document.getElementById('player-turn').className;
+  const whichCell = document.getElementById(cell);
+  if (playerTurn === '0') {
+    whichCell.innerText = '❌';
+  } else {
+    whichCell.innerText = '⭕';
+  }
+  whichCell.disabled = true;
+
+  if (cell < 4) {
```

Comment:
It would show your intentions more clearly to write it like this:

const row = Math.floor((cell - 1) / 3);
xando[playerTurn][row]++;

This way also avoids having a bunch of "if" statements.

Reply:
Done